flag/service/registry/registry.go

```diff
@@ -3,4 +3,7 @@ package registry
 type Registry struct {
 	Domain  string
 	Mirrors string
+
+	// Registry tokens
+	DockerhubToken string
```

Comment:
Shouldn't we split it here to `Dockerhub.Token` too?

Reply:
Let's not.